lib/src/dart3_suggestors/null_safety_prep/required_flux_props.dart

```diff
@@ -42,6 +42,15 @@ class RequiredFluxProps extends RecursiveAstVisitor with ClassSuggestor {
   visitCascadeExpression(CascadeExpression node) {
     final cascadeWriteEl = node.staticType?.element;
     if (cascadeWriteEl is! ClassElement) return;
+    final isReturnedAsDefaultProps = node.ancestors
+            .whereType<MethodDeclaration>()
+            .firstOrNull
+            ?.name
+            .value()
+            .toString()
```

Comment:
#nit This can be just
```suggestion
            .lexeme
```

Reply:
Addressed in https://github.com/Workiva/over_react_codemod/pull/268/commits/d36a65610c17e2d5768b344bc5ca0038efe5206d